readme.md

```diff
@@ -10,28 +10,13 @@ This is the main source code repository for Unix. It contains the source code fo
 - This bot is still in development; if any issue arises feel free to create an issue on Github.
 #### **Contributing**
 
-If you are interested in contributing to the Unix Bot:
-
-* Create a fork:
-* Clone your fork:
-```sh
-git clone https://github.com/<usernameHere>/UnixBot.git
-```
-* Create a branch:
-```sh
-git checkout -b feature/MyAmazingFeature
-```
-* Commit your changes:
-```sh
-git add *
-git commit "Commit message"
-git push origin feature/MyAmazingFeature
-```
-* Open a Pull Request!
+This README is for users and server admins, You can get more details on how to contribute [here](https://github.com/n-Ultima/UnixBot/blob/main/.github/CONTRIBUTING.md).
```

Comment:
I feel like this can be changed to something neater, like: "If you are interested in contributing, please view the [guidelines](https://www.github.com/n-Ultima/UnixBot/blob/main/.github/CONTRIBUTING.md).